subprojects/model-core/src/test/groovy/org/gradle/internal/reflect/JavaTestSubjectSubclass.java

```diff
@@ -16,6 +16,7 @@
 
 package org.gradle.internal.reflect;
 
+@SuppressWarnings("UnusedDeclaration")
```

Comment:
This would be better on the method.

Reply:
Which method? The field and a method in the class are unused. So you think I should put to annotation on each of those members?

Reply:
Ah, didn't see the field. Yes, in that case we should have an annotation on each of the unused things (assuming they are actually used in a way that IntelliJ cannot follow).

Reply:
This is a class used for testing reflection. Intellij has no chance to know the fields/methods are used. The same would be true for `JavaMethodTestSubject`, and then I would need to place that annotations an all 10 methods there and I really don't see the benefit for doing so.

Reply:
Ah, sorry, didn't recognize it was especially used for reflection. Disregard, please.